cmd/kubeadm/app/cmd/alpha/kubelet.go

```diff
@@ -38,13 +40,13 @@ var (
 		"kubelet --version" or respects the --kubelet-version parameter.
 		` + cmdutil.AlphaDisclaimer)
 
-	kubeletConfigDownloadExample = normalizer.Examples(`
+	kubeletConfigDownloadExample = normalizer.Examples(fmt.Sprintf(`
 		# Downloads the kubelet configuration from the ConfigMap in the cluster. Autodetects the kubelet version.
 		kubeadm alpha phase kubelet config download
 
 		# Downloads the kubelet configuration from the ConfigMap in the cluster. Uses a specific desired kubelet version.
-		kubeadm alpha phase kubelet config download --kubelet-version v1.12.0
-		`)
+		kubeadm alpha phase kubelet config download --kubelet-version %s
+		`, constants.MinimumKubeletVersion))
```

Comment:
@fabriziopandini @rosti @bart0sh @rojkov

instead of the usage of this constant here i think we should just have a `CurrentKubernetesMinorVersion` (or a similar naming) that is `v1.13.0`.
we can then use this version in such CLI examples and also have it as the current version for unit tests.
WDYT?

Reply:
would that `Current...` constant require manual updating on every release? that does not seem like a good idea

Reply:
we already bump these on each release:
https://github.com/kubernetes/kubernetes/blob/master/cmd/kubeadm/app/constants/constants.go#L380-L383

in an ideal world:
- an app runtime should fail if the build system did not add versioning to the binary.
- skew versions and current versions should be obtained in variables based of the client version.
- unit tests should not be bound to internet or client versioning functionality.
- a default version should not be fetched from the internet by default.

Reply:
Considering that this PR IMO is already achieving the goals to simplify version bumps, which was the original scope of the underlying issue, I'm not sure it is a good idea to conflate in this PR other goals like fixing version discovery for unit test or shifting to a dynamic approach for handling version skew policy. Might be follow up PRs are more appropriate in this case

Reply:
then again, using `constants.MinimumKubeletVersion` as an example in the CLI might introduce problems. this is a version that is currently 1.12 instead of 1.13.

Reply:
The idea of `CurrentKubernetesVersion` is good one, but outside of the scope of this PR. It can be done in some of the followup PRs though. For a start we can bump this version manually at the end of the release cycle, but ideally this should be done automatically by the build system.